ext/java/nokogiri/internals/c14n/UtfHelper.java

```diff
@@ -22,11 +22,10 @@
 import java.io.OutputStream;
 import java.util.Map;
 
-public final class UtfHelpper
+public final class UtfHelper
 {
 
-  private
-  UtfHelpper()
+  private UtfHelper()
```

Comment:
The `rake format` task will split the return type out onto a separate line again. Please preserve that newline.

Reply:
I've fixed the formatting. It looks like there are other C/Java files that don't conform to the style - at least when running Artistic Style 3.4.15.